### ISSUE TITLE
functional_writing not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#993]
https://rules.abaplint.org/functional_writing/

```abap
CALL METHOD lo_http_utility->decode_base64
  EXPORTING
    encoded = request->get_inner_rest_request( )->get_header_field( iv_name = 'Password' )
  RECEIVING
    decoded = lv_password.
```
